python/cudf/cudf/tests/test_parquet.py

```diff
@@ -4477,3 +4477,17 @@ def test_parquet_decompression(set_decomp_env_vars, my_pdf, compression):
     got = cudf.read_parquet(buffer)
 
     assert_eq(expect, got)
+
+
+def test_parquet_reader_many_large_lists():
```

Comment:
Takes roughly 4.06s to run on dgx05 machine. Can consider removing it.

Reply:
Is it faster if you use int8 instead of float types? Less memory to move.

Reply:
Good idea. Let me try that. I had just used the original use case where this error first showed up.

Reply:
Improved speed to 3.15s with booleans. Updated the test

Reply:
Three seconds is still significant IMO.
What are the chances that this kind of issue will reemerge if we don't include the test?

Reply:
Making this faster would be ideal, but I am okay with having this as-is. No strong feelings from me. We run Python tests in parallel with workstealing, so having a finite number of slow tests that test large-scale behavior isn't too worrisome to me.

Reply:
Moved this test to C++. Ran on my workstation (RTX 5880 instead of V100) and it completes in 202ms